src/main/java/sparksoniq/jsoniq/runtime/iterator/functions/DynamicFunctionCallIterator.java

```diff
@@ -23,6 +23,7 @@
 import org.apache.spark.api.java.JavaRDD;
 import org.rumbledb.api.Item;
 import sparksoniq.exceptions.IteratorFlowException;
+import sparksoniq.exceptions.SequenceExceptionExactlyOne;
```

Comment:
Is this needed?

Reply:
No, probably just something added by mistake. 
Thank you